antiplag/tasks.py

```diff
@@ -1,19 +1,61 @@
 from celery import shared_task
-from models import *
-from text_comparison import text_comparison
-import json
+
+from .models import Result
+
+from nlp.text_comparison import text_comparison
+from nlp.elastic import Elastic
+
+
+THRESHOLD = 0.15
+SIMILAR_COUNT = 10
+ERROR_MESSAGE = "An error occured during similarity comparison"
+
 
 @shared_task(name='antiplag.tasks.generate_result')
-def generate_result(document, elastic_result):
-    # One document to check against,
-    #up to 10 documents from elastic
-    corpus = document
-    corpus.append(elactic_result)
-    try:
-        result_json = text_comparison(document, corpus)
-    except:
-        antiplag_result = Result(document_id=document, error_msg="Comparison failed")
-        aantiplag_result.save()
-
-    antiplag_result = Result(document_id=document, matched_docs=result_json)
-    aantiplag_result.save()
+def compare_documents(documents):
+    """
+    Compare given documents against each other and N most similar documents from elastic
+    """
+
+    for doc in documents:
+        # returns list of dictionaries
```

Comment:
Maybe putting this into block comment?

Reply:
Not sure what you mean. It is block comment.

Reply:
I meant whether the string comment `""""""` wouldn't be better for multiline comments.

Reply:
It is reserved for docstrings or multiline strings. Convention is to use # - https://www.python.org/dev/peps/pep-0008/#block-comments

Reply:
Good to know. Thanks